Shared/Extensions/StringExtensions.swift

```diff
@@ -109,4 +109,15 @@ public extension String {
         addWordsDescriptionBolded.setAttributes(attributes, range: nsRangeOfBoldedText)
         return addWordsDescriptionBolded
     }
+    
+    public func jsonObject() -> Any? {
+        do {
+            if let data = self.data(using: .utf8) {
+                return try JSONSerialization.jsonObject(with: data, options: .mutableContainers)
+            }
+        } catch {}
```

Comment:
Since you are not using the catch you could refactor this function to use a `try?` If an error is thrown it will just return `nil`

```swift
guard let data = self.data(using: .utf8) else { return nil }
return try? JSONSerialization.jsonObject(with: data, options: .mutableContainers)
```

Reply:
Hi, thanks reviewing this pr. Unfortunately this pr is still WIP and in draft, there are lot more changes that will be pushed out soon. 
In this case I have added a logger, hence we need to keep the implementation.

Reply:
Gotcha